llvm/include/llvm-c/CAS/PluginAPI_functions.h

```diff
@@ -164,13 +164,17 @@ LLCAS_PUBLIC llcas_digest_t llcas_objectid_get_digest(llcas_cas_t,
 /**
  * Checks whether a \c llcas_objectid_t points to an existing object.
  *
+ * \param globally if true it is a hint to the underlying implementation that
```

Comment:
I think this needs to be more clearly defined but I am not sure about the correct wording for it.
```
CAS implementation doesn't have to lookup remotely if globally is not set to true, and will return lookup not found, even the value might exists in the remote CAS.
```
or it might even imply:
```
CAS implementation doesn't have to lookup or synchronized with remote. It can return false on any lookup with an object id that is not previously stored or looked up locally already.
```

Reply:
Thank you for the suggestions! Updated the doc-comment here: https://github.com/apple/llvm-project/commit/1e43d4215893a55ae3152bcc96fc209b68febfbb